templates/security/certifications.html

```diff
@@ -139,11 +139,51 @@ <h2>How does Ubuntu enable your compliance with FedRAMP, FISMA, FIPS, and DISA-S
     </div>
   </section>
 
+  <section class="p-strip">
+    <div class="row">
+      <div class="col-6">
+        <h2>Maximizing security and compliance in the US public sector with Ubuntu Pro</h2>
+        <p>
+          Navigating the maze of complex compliance requirements facing the US Public Sector is a daunting prospect. Confusing abbreviations and terminology only make charting this course more difficult. If you’re looking to understand what FIPS, FedRAMP and DISA-STIG are all about, this whitepaper is for you.
+        </p>
+        <a href="/engage/security-compliance-US-public-sector-FIPS-DISASTIG"
+           class="p-button--positive js-invoke-modal">Download the whitepaper</a>
+      </div>
+      <div class="col-6 u-hide--medium u-hide--small u-align--center">
+        <img src="https://assets.ubuntu.com/v1/4f295d2e-Whitepaper-Containers-Security.png"
+             alt="Whitepaper-Containers-Security"
+             loading="lazy" />
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-6">
+        <h2>A guide to Infrastructure Hardening</h2>
+        <p>
+          The ever-present threats of ransomware and data breaches make it imperative to lock down systems and prevent attackers from gaining a foothold. Using industry best-practice guidelines such as the CIS benchmarks, this whitepaper will walk you through the process of hardening Linux-based deployments.
+        </p>
+        <a href="/engage/a-guide-to-infrastructure-hardening"
+           class="p-button--positive js-invoke-modal">Download the whitepaper</a>
```

Comment:
```suggestion
           class="p-button--positive js-invoke-modal">Download the whitepaper</a>
```
We don't want to invoke the modal here, as this is a link to another page/form